Changelog for v2.160.0 (#3511)
vertexproject/synapse: CHANGELOG.rst

  ``it:exec:mutex``
    This now inherits the ``it:host:activity`` interface.

  ``it:exec:pipe``
    This now inherits the ``it:host:activity`` interface.

  ``it:exec:url``
    This now inherits the ``it:host:activity`` interface.

  ``it:exec:bind``
    This now inherits the ``it:host:activity`` interface.

  ``it:exec:file:add``
    This now inherits the ``it:host:activity`` interface.

  ``it:exec:file:read``
    This now inherits the ``it:host:activity`` interface.

  ``it:exec:file:write``
    This now inherits the ``it:host:activity`` interface.

  ``it:exec:file:del``
    This now inherits the ``it:host:activity`` interface.

  ``it:exec:reg:get``
    This now inherits the ``it:host:activity`` interface.

  ``it:exec:reg:set``
    This now inherits the ``it:host:activity`` interface.

  ``it:exec:reg:del``
    This now inherits the ``it:host:activity`` interface.

Features and Enhancements
-------------------------
- Add tab completion of commands, forms, properties, tags, and ``$lib.``
  functions the Storm CLI tool.
  (`#3493 <https://github.com/vertexproject/synapse/pull/3493>`_)
  (`#3507 <https://github.com/vertexproject/synapse/pull/3507>`_)

> - Add a ``node.set.<form>.<prop>`` and ``node.del.<form>.<prop>`` permissions

Reviewer: ```suggestion
- Add ``node.set.<form>.<prop>`` and ``node.del.<form>.<prop>`` permissions
```

> - Add a ``node.set.<form>.<prop>`` and ``node.del.<form>.<prop>`` permissions
>   convention to the Cortex for property

Reviewer: ```suggestion
- Add ``node.set.<form>.<prop>`` and ``node.del.<form>.<prop>`` permissions
  conventions to the Cortex for property sets and deletes.
```

  (`#3507 <https://github.com/vertexproject/synapse/pull/3507>`_)
- Add ``node.set.<form>.<prop>`` and ``node.del.<form>.<prop>`` permissions
  conventions to the Cortex for property sets and deletes.
  (`#3505 <https://github.com/vertexproject/synapse/pull/3505>`_)
- Add experimental support for Storm query offloading to the Cortex. This can

>   be used to offload Storm queries to an Aha service pool. This can be

Reviewer: ```suggestion
  be used to offload Storm queries to an AHA service pool. This can be
```
